### PR TITLE
feat: get repository contributors last activities

### PR DESCRIPTION
**Pull request by Rossb0b**

### CODE REVIEW

**Comment by Rossb0b** [comment]
⚠️ Taking around 0.8s to fetch data for a repository like "nodejs/undici" (about 30 contributors)

File: index.js [hunk 1]
```diff
@@ -78,6 +79,73 @@ export async function downloadAndExtract(repository, options = Object.create(nul
   return result;
 }
 
+export async function getContributorsLastActivities(owner, repository, options = Object.create(null)) {
+  if (typeof owner !== "string") {
+    throw new TypeError("owner must be a string!");
```
Comment (tony-go) [inline index.js:84]: WDYT about:

```suggestion
    throw new TypeError(`owner must be a string, but got ${owner}`);
```

File: index.js [hunk 2]
```diff
@@ -78,6 +79,73 @@ export async function downloadAndExtract(repository, options = Object.create(nul
...
+  const { data } = await httpie.get(contributorsUrl, {
+    headers: {
+      "User-Agent": "NodeSecure",
+      Authorization: typeof token === "string" ? `token ${token}` : GITHUB_TOKEN
+    },
+    maxRedirections: 1
+  });
+
+  const lastActivities = new Map();
+  async function hydrateLastActivities(contributor) {
```
Comment (tony-go) [inline index.js:104]: To increase readability, WDYT about putting `hydrateLastActivities` outside of this function?

**Review by tony-go — APPROVED** [review]
lgtm

File: index.js (hunk 2 above)
Comment (Rossb0b) [reply index.js]: Is this good?

**Comment by fraxken** [comment]
@all-contributors please add @Rossb0b for code

**Pull request merged**